tests/unit/composables/QuasarFieldRule.spec.js

```diff
@@ -236,4 +195,16 @@ describe('Test composable: InputRule', () => {
       }], 'plugin1', 'test1')).toEqual(key);
     });
   });
+
+  describe('Test function: isUnique', () => {
```

Comment:
to put in commit  when you add the function

Reply:
done